Demo/MozillaSocial-iOS/MoSoContent.xcodeproj/project.pbxproj

```diff
@@ -163,6 +167,7 @@
 		AA7BDC972AB2C38C00ACCACF /* Config */ = {
 			isa = PBXGroup;
 			children = (
+				8AA6BA342AC491E000B93B6C /* secrets.xcconfig */,
```

Comment:
I had issues with the project failing due to secrets file. Should I revert this change or add it here?